ReSharper.FSharp/src/FSharp.Psi.Features/src/Util/PsiUtil.fs

```diff
@@ -305,6 +307,22 @@ module PsiModificationUtil =
             Whitespace(indent)
         ] |> ignore
 
+    let private toModuleNamespaceDeclaration
+            tokenType
+            (nodeType: CompositeNodeType)
+            (moduleDeclaration: IDeclaredModuleLikeDeclaration) =
+        if moduleDeclaration == null then () else
+
+        replaceWithToken moduleDeclaration.ModuleOrNamespaceKeyword tokenType
+        let newModuleDeclaration = ModificationUtil.ReplaceChild(moduleDeclaration, nodeType.Create())
+        LowLevelModificationUtil.AddChild(newModuleDeclaration, moduleDeclaration.Children().AsArray())
+
+    let convertModuleToNamespace: IDeclaredModuleLikeDeclaration -> unit =
```

Comment:
Please move it to a separate util module, it doesn't seem generic enough to be next to `replaceWithCopy` or `deleteChild` helpers.